src/MapIterator.php

```diff
@@ -55,7 +55,7 @@ public function __construct($iterator, callable $callable)
      * Alters the current item with $this->callable and returns a new item.
      * Be careful with your types as we can't do static type checking here!
      *
-     * @return mixed
+     * @return mixed|null
```

Comment:
Mixed can be null to? Can we revert to:

```suggestion
     * @return mixed
```